MAINTAINERS.md

```diff
@@ -0,0 +1,197 @@
+
+The title of Maintainer in Bitcoin is more of a "janitorial responsibility"
+than a position of privilege. However, the role does include permissions on the
+repository, and connotes a general sense of stewardship over the project.
+
+
+Largely, the role of maintainers is to decide to merge Pull Requests. This
+process is described in [CONTRIBUTING.md](/CONTRIBUTING.md), but as an excerpt:
+
+> Whether a pull request is merged into Bitcoin Core rests with the project merge
+> maintainers.
+>
+> Maintainers will take into consideration if a patch is in line with the general
+> principles of the project; meets the minimum standards for inclusion; and will
+> judge the general consensus of contributors.
+
+Thus maintainers role is not directly elevated above that of regular
+contributors, as maintainers serve mostly as arbiters of if changes are
+acceptable to the contributors to the project in aggregate.
+
+The role of the Maintainer is currently evolving with respect to "Scoped
+Maintainers". The concept of a scoped maintainer is that it is helpful to add
+maintainers who are subject matter experts in a particular area of the code,
+but perhaps not in other areas. As such, maintainers with an explicit scope
+typically will avoid merging PRs in areas outside their scope, but may do so on
+occasion. As an individual maintainer's experience levels in the project
+increase, they may change their focus from scoped to general, but this is not
+currently an explicit designation (all maintainers should be considered to be
+general at this time). As a rough measure, scoped maintainers should avoid
+"stepping on the toes" of work that is being guided by another maintainer or
+contributor.
+
+Historically, Bitcoin has also had a "Lead Maintainer", which was most recently
+@laanwj. However, [in a blog
+post](https://laanwj.github.io/2021/01/21/decentralize.html), @laanwj stated
+that they wanted to take more of a background role as a maintainer and have not
+"passed the baton" to a successor as a Lead Maintainer, and instead focused on
+decentralizing the maintainership ecosystem. As such, while the title of Lead
+Maintainer still is held by @laanwj, it should be considered a bug-not-feature
+if it confers any additional responsibility or privilege over other General
+Maintainers, and may be explicitly deprecated in the future.
+
+## Self Merges
+
+It is discouraged for maintainers to merge their own work. However, given that
+there may not be another maintainer who regularly works on the section of code
+or is familiar, self-merge is not forbidden. However, as a guard, self merges
+should usually only happen if the following criteria has been met:
+
+1. The PR has been reviewed and ACK'd by other regular contributors familiar
+   with that section of the code.
+1. There are no NACKs being overruled on that PR.
+1. The PR is at least 1 week old, to permit time for review.
+1. The maintainer should communicate intent to self-merge 48 hours before doing so
```

Comment:
I don't see a reason why self-merges should be treated any different than regular merges. In fact, if you do treat them differently, you will likely achieve the opposite of what you want to achieve. If there is a requirement for maintainers to ask other maintainers for a favour to "take a look" at a pull request to avoid a self-merge, I'd be worried that this will decrease transparency and instead create an incentive for maintainers to swap merges of each other's pulls.



See also https://github.com/bitcoin/bitcoin/pull/25524#issuecomment-1172838909 and https://github.com/bitcoin/bitcoin/pull/25524#issuecomment-1173806702

It might be better if you stated your concern/goal and then work on a solution that is better suited. For example, if your concern is that maintainers are too productive, I think it would be more helpful to find more maintainers to share their workload.

Reply:
I have shared two reasons why self merges should be discouraged: https://github.com/bitcoin/bitcoin/pull/25560#discussion_r915645296

Reply:
It's not clear if this document is that place to state concerns, or to just state the policy as a result of the concerns.